src/zopfli-plugin.js

```diff
@@ -13,26 +13,52 @@ const defaultOptions = {
 
 const globAsync = util.promisify(glob)
 
-async function onPostBuild (args, pluginOptions) {
+async function onPostBuild ({ reporter }, pluginOptions) {
   const options = { ...defaultOptions, ...pluginOptions }
   const fileBasePath = path.join(process.cwd(), 'public')
   const patternExt = (options.extensions.length > 1) ? `{${options.extensions.join(',')}}` : options.extensions[0]
   const pattern = `**/*.${patternExt}`
 
   const files = await globAsync(pattern, { cwd: fileBasePath, ignore: '**/*.gz', nodir: true })
-  const tmrStart = new Date().getTime()
 
   const compressFile = workerFarm(worker)
+
+  const activity = reporter.activityTimer('Zopfli compression')
+  activity.start()
+
+  let totalCompressed = 0
+  let totalSavings = 0
   const compress = files.map(file => {
     return new Promise((resolve, reject) => {
-      compressFile(file, pluginOptions, err => err ? reject(err) : resolve())
+      compressFile(file, pluginOptions, (details, err) => {
```

Comment:
You've specified some extra zopfli options to `compressFile`. But this function, that is in the `workerFarm` does not pick them up from the `pluginOptions`. Maybe you've forgot to push the changes in that file?

Reply:
You are right, I did not commit that file.  Will update